OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -81,6 +81,23 @@ extension OmnipodPumpManagerError: LocalizedError {
     }
 }
 
+extension BinaryInteger {
```

Comment:
This extension should be fileprivate in the same file responsible for decoding the pulse log (PodInfoPulseLogRecent.swift).